pkg/client/connection.go

```diff
@@ -17,6 +22,7 @@ import (
 
 type ApiConnectionDetails struct {
 	ArmadaUrl                   string
+	ArmadaRestUrl               string
```

Comment:
Is it possible to just construct the ArmadaRestUrl from ArmadaUrl?  

I don't love a new config value that requires our client users to put both a RestURL and a grpc URL.

Reply:
No. They may be completely different.

Reply:
So how does someone understand what to put for ArmadaRestUrl?  

This is a config that our users of armadactl and other client libs use so I don't understand how they would know that the RestURL that grpc is generating could be very different from the connection details they would use for the internal grpc server.

Reply:
@kannon92 grpc and rest could quite possibly be on completely different (sub)domains, where the REST server exposes "safe" endpoints and could be made public, and the gRPC server exposes "unsafe/admin" endpoints, and could be made private.